### PR TITLE
{vis}[foss/2020b] pymca v5.6.3, silx v0.14.0, PyOpenGL v3.1.5, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#12231~~ (PyOpenCL)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3551.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b1f0353204823edfca8b626cee3e77cf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node2614.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/5c72ef329ccdafa90253790e8a517f21 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/036da51948a69d72c28570df3aa5300d for a full test report.

**Comment by branfosj** [comment]
close / reopen to retrigger tests now that  #12231 is merged

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4da6a9d96e94cdd6990b80ed0b32887f for a full test report.

File: easybuild/easyconfigs/q/Qtconsole/Qtconsole-5.0.2-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+name = 'Qtconsole'
+version = '5.0.2'
+
+homepage = 'https://jupyter.org/'
+description = """A rich Qt-based console for working with Jupyter kernels, supporting rich media output, session export, and more.
+The Qtconsole is a very lightweight application that largely feels like a terminal, 
+but provides a number of enhancements only possible in a GUI, 
+such as inline figures, proper multiline editing with syntax highlighting, graphical calltips, and more."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```
Comment (branfosj) [inline easybuild/easyconfigs/q/Qtconsole/Qtconsole-5.0.2-foss-2020b.eb:12]:
```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

We've moved IPython to GCCcore, so this should be moved as well.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/q/Qtconsole/Qtconsole-5.0.2-foss-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/q/Qtconsole/Qtconsole-5.0.2-foss-2020b.eb]: fixed in dae3e66368

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7513a4350e806ae5151bd1a1b7d0547f for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**